pkg/runtime/kubeadm.go

```diff
@@ -113,15 +113,21 @@ func (k *KubeadmRuntime) getCGroupDriver(node string) (string, error) {
 }
 
 func (k *KubeadmRuntime) MergeKubeadmConfig() error {
+	k.ImageKubeVersion = k.getKubeVersionFromImage()
+	for _, fn := range []string{
+		"",                          // generate default kubeadm configs
+		k.getDefaultKubeadmConfig(), // merging from predefined path of file if file exists
```

Comment:
1. k.getDefaultKubeadmConfig()
2. “”

is OK

Reply:
This way in the comment, as well as the original, both are setting default values first, then dst will not be overwritten even if the src is set. IMO, with mergo's transformers is more intuitive.